cumulusci/core/keychain/tests/test_base_project_keychain.py

```diff
@@ -247,6 +247,31 @@ def test_list_services(self, keychain):
         assert services["devhub"] == ["bar_alias", "foo_alias"]
         assert services["github"] == ["zed_alias", "zoo_alias"]
 
+    def test_get_services_for_type(self, keychain):
+        service_config = ServiceConfig({"foo": "bar"})
+        keychain.services = {
+            "devhub": {
+                "foo_alias": service_config,
+                "bar_alias": service_config,
+            },
```

Comment:
Now that I see the test I'm realizing that we could just access the service directly with `keychain.services[service_type]`. Whoops.